<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QWidget" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>960</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>累计时间：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="sended_text">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>551</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="error_byte">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>210</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>    误码率：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="received_byte">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="start_time">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>880</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="sended_byte">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>591</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>接收数据：</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_send_byte_">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>290</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>需发送字节：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>330</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>已发送字节：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>IP设置</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>   Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="port">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>8888</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>    I P:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="server_ip">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.246</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="connect">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="recevied_text">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>551</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="total_send_byte">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>370</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>已接收字节：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>920</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>结束时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_byte_">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>250</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>  错误字节：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>880</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始时间：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cumulative_time">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>960</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="end_time">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>920</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="error_rate_">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>591</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>发送数据：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>误码率统计：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>850</y>
     <width>231</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>时间记录：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>清零</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>自动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>520</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>清零</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>手动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkout">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>590</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>手动校验</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>校验</string>
   </property>
  </widget>
  <zorder>groupBox_8</zorder>
  <zorder>groupBox_7</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_6</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_5</zorder>
  <zorder>recevied_text</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>label_4</zorder>
  <zorder>sended_text</zorder>
  <zorder>error_byte</zorder>
  <zorder>label_10</zorder>
  <zorder>received_byte</zorder>
  <zorder>start_time</zorder>
  <zorder>sended_byte</zorder>
  <zorder>total_send_byte_</zorder>
  <zorder>label_7</zorder>
  <zorder>start</zorder>
  <zorder>total_send_byte</zorder>
  <zorder>label_8</zorder>
  <zorder>label_6</zorder>
  <zorder>error_byte_</zorder>
  <zorder>label_9</zorder>
  <zorder>cumulative_time</zorder>
  <zorder>end_time</zorder>
  <zorder>error_rate_</zorder>
  <zorder>reset</zorder>
  <zorder>reset_2</zorder>
  <zorder>start_2</zorder>
  <zorder>checkout</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
